lte/gateway/deploy/roles/magma/files/config_stateless_utils.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
```

Comment:
should we consider using python for these scripts at this point? there's a lot of code you could just re-use here, config loading in python already handles overrides. Or do these get executed before the python interpreter is set up?

Reply:
These scripts are used to turn on/off stateless AGW, which can be run after the python interpreter. But the stateless mode needs changes to both Yaml configs and systemd unit files so we do need to execute a bunch of shell commands, which is the reason I started with shell script in the first place. Can you point me to the config loading python script? May be I can wrap python calls within the shell script.

Reply:
orc8r/gateway/python/magma/configuration/service_configs.py has the YML config loading stuff you're probably looking for